flow/project.py

```diff
@@ -701,9 +701,9 @@ def foo_label(job):
         The ``foo-label-text`` label will now show up in the status view for each job,
         where the ``foo`` key evaluates true.
 
-        If instead of a ``str``, the label functions returns any other type, the label
-        name will be the name of the function if and only if the return value evaluates
-        to ``True``, for example:
+        If the label functions returns any type other than ``str``, the label
+        name will be the name of the function if and only if the return value
+        evaluates to ``True``, for example:
```

Comment:
While we're here, could you also add a proper parameter docstring?

Reply:
I added a parameter docstring but I'm not sure if it's accurate. I don't fully understand _how_ this decorator resolves between name/function, but that's how I documented it.